<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>775</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Book Antiqua</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>301</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Settings Reglin</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="y_">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>160</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>120</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Target</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>130</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Test Size</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="test_size_">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>160</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>781</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>781</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="run">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>240</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>RUN</string>
    </property>
   </widget>
   <widget class="QTableView" name="coeff_view">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>371</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="mae_view">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>320</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="mse_view">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>410</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="rmse_view">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>510</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="mae">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>340</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>MAE :</string>
    </property>
   </widget>
   <widget class="QLabel" name="mse">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>430</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>MSE :</string>
    </property>
   </widget>
   <widget class="QLabel" name="rmse">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>530</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>RMSE :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="settings_name">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>670</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="save_settings">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>710</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Settings</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>630</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Settings Name</string>
    </property>
   </widget>
   <widget class="QPushButton" name="back">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>61</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QPushButton" name="main_menu">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>10</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Main Menu</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
